### PR TITLE
[utility], [bitset] Introduce a separate heading for the synopsis.

### PR DESCRIPTION
**Pull request by jensmaurer**
Partially addresses #566.

### CODE REVIEW

File: source/utilities.tex [hunk 1]
```diff
@@ -40,7 +40,7 @@
 \indextext{\idxhdr{utility}}%
 \indexlibrary{\idxhdr{utility}}%
 \indexlibrary{\idxcode{rel_ops}}%
-\synopsis{Header \tcode{<utility>} synopsis}
+\rSec2[utility.syn]{Header \tcode{<utility>} synopsis}
```
Comment (tkoeppe) [inline source/utilities.tex]: I had thought about this, too, but until now no "Synopsis" section that has a numbered heading has an introductory paragraph. That's why I haven't made this change yet, I wonder if we need some more rewording.

Comment (tkoeppe) [inline source/utilities.tex]: Right, with this change we would have a numbered section called "Synopsis" that does *not* start with a code block. That'd be new, and moreover it seems that all these introductory sentences don't really add any normative value. So maybe we could revise this on at a slightly larger scale?

Comment (jensmaurer) [inline source/utilities.tex]: Ah, this is talking about <utility>, which somehow made it into the [bitset] patch.

**Comment by jensmaurer** [comment]
Ok, I've moved the introductory comment for <utility> at the bottom of the synopsis. This is the same as for bitset.

File: source/utilities.tex (hunk 1 above)
Comment (tkoeppe) [inline source/utilities.tex]: Yeah, maybe -- I have never managed to make up my mind about this one. (And I have looked at this case before, and others awkward cases like `<algoritms>`.) Not sure what the best way is to present this.

Comment (jensmaurer) [inline source/utilities.tex]: This interacts somewhat with #574. If we remove those "in general" clauses everywhere and move their contents to level x-1, the introductory sentence for &lt;utility> can simply go at that level, too.

**Pull request merged**